synapse/tests/test_cmds_cron.py

```diff
@@ -120,8 +120,6 @@ async def getNextFoo():
                     self.true(outp.expect('Modified cron job'))
                     await cmdr.runCmdLine(f"cron edit xxx {{[graph:node='*' :type=m2]}}")
                     self.true(outp.expect('does not match'))
-                    await cmdr.runCmdLine(f"cron mod xxx yyy")
```

Comment:
is this now a valid cron command?

Reply:
Yep! Because the argument can be a string or an argv query.